homeassistant/components/snapcast/media_player.py

```diff
@@ -31,9 +36,13 @@
 CLIENT_SUFFIX = 'Snapcast Client'
 
 SERVICE_SCHEMA = vol.Schema({
-    ATTR_ENTITY_ID: cv.entity_ids,
+    vol.Required(ATTR_ENTITY_ID): cv.entity_ids,
 })
 
+JOIN_SERVICE_SCHEMA = SERVICE_SCHEMA.extend({
+    vol.Required(ATTR_MASTER): cv.entity_id,
```

Comment:
Why not use `ATTR_ENTITY_ID` to make it clear that it's just another entity, and no special value?

Reply:
I used `ATTR_MASTER` to distinguish the clients moving to the master's group and the master not moving from his group. 
So after the `join`, all clients are playing the master's stream